contracts/token/ERC20/utils/SafeERC20.sol

```diff
@@ -79,6 +80,25 @@ library SafeERC20 {
         }
     }
 
+    function safePermit(
+        IERC20Permit token,
+        address owner,
+        address spender,
+        uint256 value,
+        uint256 deadline,
+        uint8 v,
+        bytes32 r,
+        bytes32 s
+    ) internal {
+        uint256 nonceBefore = token.nonces(owner);
+        _callOptionalReturn(
```

Comment:
This doesn't make sense to me. `permit` doesn't have a return value, so there is no optional return. We should simply call `permit` directly.